atc/exec/set_pipeline_step.go

```diff
@@ -130,7 +130,40 @@ func (step *SetPipelineStep) run(ctx context.Context, state RunState) error {
 		return nil
 	}
 
-	team := step.teamFactory.GetByID(step.metadata.TeamID)
+	var team db.Team
+	if step.plan.Team == "" {
+		team = step.teamFactory.GetByID(step.metadata.TeamID)
+	} else {
+		fmt.Fprintln(stderr, "\x1b[1;33mWARNING: specifying the team in a set_pipeline step is experimental and subject to change!\x1b[0m")
```

Comment:
I think we should use even stronger scare words here:

```suggestion
		fmt.Fprintln(stderr, "\x1b[1;33mWARNING: specifying the team in a set_pipeline step is experimental will likely be removed in the future!\x1b[0m")
```

Reply:
Force pushed so the wording is now as you suggest